Networking/Apiary and API call mocking.md

```diff
@@ -0,0 +1,179 @@
+## What is Apiary? 
+
+Apiary is that tool that lets you build your API extremely fast, allows you to build functional API in a single day, sometimes even a few hours. 
+
+## How does it work?
+
+You can look at Apiary like an online collection of `JSON` responses with a bit more power. You get some base URL and all you have to do is define `endpoints` and their `response`. Example of one such response can be seen below :
+
+```json
+
+## Some API Endpoint [/apiEndpoint]
+
+### GET some information [GET]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "information": "Favourite programming language?",
+                "published_at": "2015-08-05T08:40:51.620Z",
+                "choices": [
+                    {
+                        "choice": "Swift",
+                        "votes": 1
+                    }
+                ]
+            }
+        ]
+
+### POST some information [POST]
+
++ Response 201 (application/json)
+
+    + Headers
+
+            Location: /collection/2
+
+    + Body
+
+            {
+                "information": "Favourite programming language?",
+                "published_at": "2015-08-05T08:40:51.620Z",
+                "choices": [
+                    {
+                        "choice": "Swift",
+                        "votes": 2
+                    } 
+                ]
+            }
+
+```
+Apiary usually uses `API Blueprint`, a powerful high-level API description language for web APIs, more details on that can be found [here](https://apiblueprint.org).
+
+
+## Why Apiary ?
+
+Here are some of reasons why you might want to add apiary to your project: 
+
+* **Easy to learn**, not just for developers, but for testers also
+
+* **Multiplatform sync**, once written API can be used for iOS, Android or Web 
+
+* **Time-saving**, takes only **1** person to setup mock for multiple platforms
+
+* **Documentation!** As you write your responses, apiary will automatically generate great documentation
+
+* **Traffic inspector** - one click away from all of the API traffic
+
+* **Easy changes**, no need to rebuild project to apply changes, just edit apiary
+
+* **Improves testability**, QA can go through most test cases without support of the backend
+
+* Because it is **awesome**
+  
+
+## Know your limits
+
+Here are some of the limitations of apiary:
+
+* **You need network connection** - no internet, no mocking
+
+* **Backend Speed**, Apiary is usually very fast so developers might get a wrong feeling about some response times
```

Comment:
Use dash rather than comma for separating keyword and description, just like for the first bullet